api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -220,6 +220,22 @@ message AccessLogCommon {
   // failure reason from the transport socket. The format of this field depends on the configured downstream
   // transport socket. Common TLS failures are in :ref:`TLS trouble shooting <arch_overview_ssl_trouble_shooting>`.
   string downstream_transport_failure_reason = 28;
+
+  // For HTTP: Total number of bytes sent to the downstream by the http stream.
+  // For TCP: Total number of bytes sent to the downstream by the tcp proxy.
+  uint64 downstream_wire_bytes_sent = 29;
+
+  // For HTTP: Total number of bytes received from the downstream by the http stream. Envoy over counts sizes of received HTTP/1.1 pipelined requests by adding up bytes of requests in the pipeline to the one currently being processed.
```

Comment:
This is a bit weird, what is the history here? Is there a GH issue to reference?

Reply:
I agree. This was copied from the [existing docs](https://github.com/envoyproxy/envoy/blame/main/docs/root/configuration/observability/access_log/usage.rst#L323-L328).

There is a bit [more context in the originating PR](https://github.com/envoyproxy/envoy/pull/17521#discussion_r729819825).

Reply:
OK. Let's keep for now then, I would suggest a reference to the PR given how weird it is. @yanavlasov should there be an issue to track this?